go/runtime/ops_publisher.go

```diff
@@ -90,6 +91,24 @@ func (p *OpsPublisher) PublishStats(
 	return err
 }
 
+// PublishConnectorLog is just like PublishLog, except it sanitizes the
+// `eventType` field of any logs, in order to prevent connectors from publishing
+// any events other than `connectorStatus`.
+func (p *OpsPublisher) PublishConnectorLog(out ops.Log) {
```

Comment:
We should do this filtering / rejection closer to the connector for two reasons:

* At this point, the log has been processed by the in-process Rust runtime service and then been sent to Go.
* I intend to reduce the amount of Go runtime code as much as possible over time.

One place to filter would be `flow-connector-init`, but that's inside of the container boundary, so probably not a great idea. `crates/runtime/container.rs` has a log loop that reads structured logs from `flow-connector-init` and dispatches to `log_handler`. Probably that's the place to filter.